web-ng/karma.conf.js

```diff
@@ -0,0 +1,32 @@
+// Karma configuration file, see link for more information
```

Comment:
Please add license header in all JS files. Example:
https://github.com/google/ground-platform/blob/master/web/src/datastore.js

(Use 2019 for year)

Reply:
Added, what about TS files?